### ISSUE TITLE
An error occurred while Style/SafeNavigation cop was inspecting …

### ISSUE THREAD

**Issue by LeEnno** [rubocop/rubocop#14501]
In version 1.79.2 and above I get the following error:

```
1 error occurred:
An error occurred while Style/SafeNavigation cop was inspecting /path/to/test.rb:3:4.
Errors are usually caused by RuboCop bugs.
Please, update to the latest RuboCop version if not already in use, and report a bug if the issue still occurs on this version.
https://github.com/rubocop/rubocop/issues

Mention the following information in the issue report:
1.79.2 (using Parser 3.3.9.0, rubocop-ast 1.46.0, analyzing as Ruby 3.1, running on ruby 3.1.4) [arm64-darwin24]
```

test.rb:

```ruby
class Foo
  def bar
    baz.nil? ? nil : baz[0].bla
  end
end

```

--------

## Expected behavior

Have rubocop not produce an error.

## Actual behavior

Rubocop produces an error.

## Steps to reproduce the problem

Execute `rubocop /path/to/test.rb` on the command line.

## RuboCop version

```
1.79.2 (using Parser 3.3.9.0, rubocop-ast 1.46.0, analyzing as Ruby 3.1, running on ruby 3.1.4) [arm64-darwin24]
  - rubocop-minitest 0.38.1
  - rubocop-performance 1.25.0
  - rubocop-rails 2.33.3
```

**Comment by Earlopain** [comment]
Thanks for the report! This is a duplicate of https://github.com/rubocop/rubocop/issues/14477 and will be fixed in the next release.

**Comment by LeEnno** [comment]
Oh, sorry. I only searched for open issues beforehand. Thank you and keep up the good work 💪
